src/sentry/static/sentry/app/components/events/errors.tsx

```diff
@@ -90,46 +122,41 @@ class EventErrors extends React.Component<Props, State> {
     }
   }
 
-  async fetchReleaseArtifacts(query: string) {
-    const {api, orgSlug, event, projectSlug} = this.props;
-    const {release} = event;
-    const releaseVersion = release?.version;
-
-    if (!releaseVersion || !query) {
-      return;
-    }
-
-    try {
-      const releaseArtifacts = await api.requestPromise(
-        `/projects/${orgSlug}/${projectSlug}/releases/${encodeURIComponent(
-          releaseVersion
-        )}/files/?query=${query}`,
-        {
-          method: 'GET',
-        }
-      );
-
-      this.setState({releaseArtifacts});
-    } catch (error) {
-      Sentry.captureException(error);
-      // do nothing, the UI will not display extra error details
-    }
-  }
-
   toggle = () => {
     this.setState(state => ({isOpen: !state.isOpen}));
   };
 
-  uniqueErrors = (errors: any[]) => uniqWith(errors, isEqual);
-
   render() {
-    const {event} = this.props;
+    const {event, hasProGuardError} = this.props;
     const {isOpen, releaseArtifacts} = this.state;
     const {dist} = event;
 
-    // XXX: uniqueErrors is not performant with large datasets
+    // return null
+
+    if (!hasProGuardError && objectIsEmpty(event.errors)) {
+      return null;
+    }
+
+    // XXX: uniqWith returns unique errors and is not performant with large datasets
     const errors =
-      event.errors.length > MAX_ERRORS ? event.errors : this.uniqueErrors(event.errors);
+      event.errors.length > MAX_ERRORS ? event.errors : uniqWith(event.errors, isEqual);
+
+    if (hasProGuardError) {
+      errors.push({
+        data: {name: 'proguard'},
+        message: tct('It seems that the [gradle] plugin was not correctly configured', {
```

Comment:
Can we link over _gradle plugin_ instead of just _gradle_ ?

Reply:
I'd prefer to be more specific, eg Sentry Gradle Plugin